src/views/SendWelcomeEmail.vue

```diff
@@ -56,7 +56,7 @@
           The Welcome Email tool automatically sends a customizable message by email to all members of your course site,
           even if the site has not yet been published. For more information, visit
           <OutboundLink href="https://berkeley.service-now.com/kb_view.do?sysparm_article=KB0013900">
-            https://berkeley.service-now.com/kb_view.do?sysparm_article=KB0013900
+            bCourses Mailing List - Welcome Email
```

Comment:
@mlincoln - Here's a nit-picky question: How would a screen-reader speak the dash? Does it make sense to word in a more natural language way?

Reply:
Boy, that _is_ a nit-picky question. I took the easy way out and re-named the KB article.